src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,171 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable/PeopleTable';
+import { PeopleFilters } from '../components/PeopleFilters/PeopleFilters';
+import { ColumnNames } from '../types/ColumnNames';
+
+interface PeopleFilterOptions {
+  people: Person[];
+  sort: string;
+  order: string;
+  query: string;
+  sex: string;
+  centuries: Array<string>;
+}
+
+const getVisiblePeople = ({
+  people,
+  sort,
+  order,
+  query,
+  sex,
+  centuries,
+}: PeopleFilterOptions) => {
```

Comment:
1. Let's put this function in separate file
2. Instead of passing filters separately, pass the searchParams object